webpack/components/extensions/RegistrationCommands/fields/Force.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'foremanReact/common/helpers';
+
+import { FormGroup, Checkbox, Popover } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const Force = ({ value, onChange, isLoading }) => (
+  <FormGroup fieldId="reg_katello_force">
+    <Checkbox
+      label={
+        <span>
+          {__('Force')}{' '}
+          <Popover
+            bodyContent={__('Remove any `katello-ca-consumer` rpms before registration and run subscription-manager with `--force` argument.')}
+          >
+            <button
+              className="pf-c-form__group-label-help"
+              onClick={e => e.preventDefault()}
+            >
+              <HelpIcon noVerticalAlign />
+            </button>
+          </Popover>
+        </span>
+      }
```

Comment:
I see that the `label` component needed to be reinvented in almost all use-cases with the form,
can you move it into foreman's common dir?